packages/fast-components-react-msft/src/action-toggle/index.ts

```diff
@@ -33,5 +34,6 @@ export {
     ActionToggleProps,
     ActionToggleClassNameContract,
     ActionToggleHandledProps,
+    actionToggleSchema,
```

Comment:
I think we should have this follow our current syntax for exports: `ActionToggleSchema`

Reply:
Talked offline decided to leave them camel cased